src/main/java/seedu/address/logic/commands/transaction/GetTotalExpenditureCommand.java

```diff
@@ -49,10 +49,9 @@ public CommandResult execute(Model model) throws CommandException {
         model.updateFilteredTransactionList(transaction -> {
             boolean isExpense = transaction.getType().type.equals(TransactionType.EXPENSE);
             boolean isSameMonth = transaction.getDateTime().getDateTime().getMonthValue() == month;
-            if (categoryFilter == null || categoryFilter.isEmpty()) {
-                return isExpense && isSameMonth;
-            }
-            return isExpense && isSameMonth && transaction.getCategory().equals(categoryFilter);
+            boolean hasCategory = categoryFilter == null
+                    || transaction.getCategories().stream().anyMatch(cat -> cat.equals(categoryFilter));
```

Comment:
Boolean condition seems tad bit long, think you can merge the null check with the equality check, so if is null, we return `true` to the `anyMatch`

Reply:
Yeah, thought it was pretty long too but intellij suggested it. Let me try to merge it into the anyMatch,  should be cleaner